lib/active_interaction/filters/file_filter.rb

```diff
@@ -4,8 +4,9 @@ module ActiveInteraction
   class Base
     # @!method self.file(*attributes, options = {})
     #   Creates accessors for the attributes and ensures that values passed to
-    #   the attributes respond to the `eof?` method. This is useful when passing
-    #   in Rails params that include a file upload or another generic IO object.
+    #   the attributes respond to the `rewind` method. This is useful when
+    #   passing in Rails params that include a file upload or another generic
+    #   IO object.
```

Comment:
Right, that makes sense. I'm concerned because we're checking methods but really we're trying to make sure a series of classes work. Maybe we're not thinking about this the right way.